content/updates/04-2020/3-connection-dashboard-updates.md

```diff
@@ -1,25 +1,24 @@
 # Connection dashboard updates
-::: meta-data type="New features" features="Connections" available="2nd of April"
+::: meta-data type="New" features="Connections, Connectivity" available="2nd of April"
 :::
 
-## Source connections charts rework
+## Source connections charts improved
 We reworked the display of the two charts about the source connections in the Connection dashboard. We also now compute the sum on the last 7 days of the number of products created and updated.
 
 ![Sources connection charts](../img/number-of-products-created-updated.png)
 
 ## Track your destination connections
```

Comment:
Add metadata here.

Reply:
added basic ones - no feature link - do you have one?